components/bpmn/bpmn-explorer-web/src/web/model/advancedFilterModel.jag

```diff
@@ -0,0 +1,237 @@
+        var qs = request.getQueryString();
+        newQueryString = qs.substring( qs.indexOf('&') + 1 );
+    } else {
+        newQueryString = request.getQueryString();
+    }
+
+    var bodyOfRequest = {};
+    var vCollection = {};
+
+    if (request.getParameter("instanceStatus") === "COMPLETED") {
+
+        searchActiveInstances = false;              
+    
+
+        if (requestParameters.processInstanceId !== undefined && requestParameters.processInstanceId.length > 0) {
+          
+            vCollection["processInstanceId"] = requestParameters.processInstanceId;
+        }
+
+        if (requestParameters.processDefinitionNameLike !== undefined && requestParameters.processDefinitionNameLike.length > 0) {
+            
+            vCollection["processDefinitionId"] = requestParameters.processDefinitionNameLike;           
+        }       
+           
+       if (requestParameters.createdBefore !== undefined && requestParameters.createdBefore.length > 0) {
+           
+            vCollection["startedBefore"] = requestParameters.createdBefore;  
+        }
+
+        if (requestParameters.createdAfter !== undefined && requestParameters.createdAfter.length > 0) {
+           
+            vCollection["startedAfter"] = requestParameters.createdAfter;  
+        }      
+       
+       if (requestParameters.variableName !== undefined && requestParameters.variableName.length > 0 && 
+            requestParameters.variableValue !== undefined && requestParameters.variableValue.length > 0) {
+
+            var variableNames = requestParameters.variableName;      
+            var values = requestParameters.variableValue;
+            
+            vCollection = {"variables":[]};
+            if (typeof variableNames == 'string') {
+
+                vCollection.variables.push({
+                            "name":variableNames,
+                            "value":values,
+                            "operation" : "equals"
+                });
+
+            } else {               
+                for (var i = 0; i < variableNames.length; i++) {
+
+                    if (variableNames[i] != null && variableNames[i] !== "") {                    
+                          vCollection.variables.push({
+                            "name":variableNames[i],
+                            "value":values[i],
+                            "operation" : "equals"
+                        });
+                               
+                    } else {
+                         break;
+                    }
+                }
+            }             
+           
+        }
+
+        vCollection["start"] = ((pagination - 1) * pageSize);
+        vCollection["size"] = pageSize;
+        vCollection["finished"] =  "true";
+        vCollection["tenantId"] =  bpsTenantID;
+        bodyOfRequest= vCollection;          
+     
+        requestUrl = httpUrl + "/bpmn/query/historic-process-instances";
+       
+
```

Comment:
As a practice don't keep 2 new lines. If you want to have a new line to separate code blocks, just use a single line